src/main/scala/Serdes.scala

```diff
@@ -505,9 +540,9 @@ class TLSerdes(w: Int, params: Seq[TLManagerParameters], beatBytes: Int = 8)
       val inDes = Module(new GenericDeserializer(mergeType, w))
       inDes.io.in <> ser.in
       tl.b.valid := inDes.io.out.valid && inDes.io.out.bits.isB()
-      tl.b.bits := TLMergedBundle.toB(inDes.io.out.bits)
```

Comment:
Nit: I wonder if there is a neater way to pass `hasCorruptDenied` around through the module hierarchy. 
Perhaps add it as a Parameters key and use 
```
p.alter({case x: HasCorruptDenied => true})
``` 
to pass the HasCorruptDenied setting into TLSerdesser

Reply:
I noticed that... but I think this is fine for now. That should be easy to do later if need be.